### PR TITLE
[dev-v.2.8] k3s August patch release

### PR DESCRIPTION
**Pull request by johnatasr**

### CODE REVIEW

**Review by brooksn — DISMISSED** [review]

**Review by dereknola — APPROVED** [review]

File: channels.yaml [hunk 1]
```diff
@@ -382,6 +382,15 @@ releases:
         type: string
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  - version: v1.24.17+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.7.99
```
Comment (dereknola) [inline channels.yaml]: This is for v2.8, so we need to bump the min and max server versions correct?

**Review by dereknola — DISMISSED** [review]

File: channels.yaml (hunk 1 above)
Comment (ShylajaDevadiga) [reply channels.yaml]: Also for all k3s versions

Comment (dereknola) [reply channels.yaml]:
```suggestion
    maxChannelServerVersion: v2.8.99
```

File: channels.yaml [hunk 2]
```diff
@@ -443,9 +458,24 @@ releases:
...
+    serverArgs: &serverArgs-v8
+      <<: *serverArgs-v7
+      tls-san-security:
+          type: boolean
+    agentArgs: *agentArgs-v3
+    featureVersions: *featureVersions-v1
+  - version: v1.27.5+k3s1
+    minChannelServerVersion: v2.7.5-alpha1
+    maxChannelServerVersion: v2.7.99
```
Comment (dereknola) [inline channels.yaml]:
```suggestion
    minChannelServerVersion: v2.8.0-alpha1
    maxChannelServerVersion: v2.8.99
```

**Review by brandond — DISMISSED** [review]

File: data/data.json [hunk 3]
```diff
@@ -20530,6 +20530,477 @@
...
+      "type": "string"
+     },
+     "system-default-registry": {
+      "type": "string"
+     }
+    },
+    "featureVersions": {
+     "encryption-key-rotation": "2.0.0"
+    },
+    "maxChannelServerVersion": "v2.7.99",
```
Comment (brooksn) [inline data/data.json:20575]:
```suggestion
    "maxChannelServerVersion": "v2.8.99",
```

**Review by brooksn — DISMISSED** [review]

**Review by briandowns — DISMISSED** [review]

**Review by briandowns — APPROVED** [review]

**Review by brooksn — APPROVED** [review]

**Pull request merged**